CHANGELOG.md

```diff
@@ -134,9 +134,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - Updated data-migration1 and data-migration2 Lambdas to use UPSERT instead of UPDATE when migrating dynamoDB records to PostgreSQL.
     - Changed data-migration1 and data-migration2 logic to only update already migrated records if the incoming record update has a newer timestamp
   - **CUMULUS-2329**
-    - Add write-db-dlq-records-to-s3 lambda.
+    - Add `write-db-dlq-records-to-s3` lambda.
     - Add terraform config to automatically write db records DLQ messages to an s3 archive on the system bucket.
     - Add unit tests and a component spec test for the above.
+  - **CUMULUS-2380**
+    - Add `process-dead-letter-archive` lambda to pick up and process dead letters in the S3 system bucket dead letter archive.
+    - Add `/deadLetters` endpoint to trigger an async operation to leverage this capability on demand.
+    - Add unit tests and integration test for all of the above.
```

Comment:
Do we have a ticket to document this feature?     I noticed this and the prior PR don't have any doc updates.

Reply:
Prior PR is automatic, but good point that this probably needs an operator doc.